ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarLegacyDemoController.swift

```diff
@@ -165,7 +165,7 @@ class AvatarLegacyViewDemoController: DemoController {
 
         let avatarContainer = UIView()
         avatarContainer.addSubview(avatarView)
-        avatarContainer.widthAnchor.constraint(equalToConstant: AvatarLegacySize.extraExtraLarge.size.width).isActive = true
+        avatarContainer.widthAnchor.constraint(equalToConstant: 72).isActive = true
```

Comment:
Why not keep it to what it was before?

Reply:
Keeping the enum size var public will make the binary larger. I think that the demo app is not enough to expose it.

Reply:
Actually I found some uses of it in devmain. I'll revert this particular change.

Reply:
Does the binary size increase with the reverted changes? If so maybe the description could be updated